.goreleaser.yml

```diff
@@ -0,0 +1,33 @@
+env:
+  - CGO_ENABLED=0
+
+builds:
+  - id: nginx-meshctl
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+      - arm64
+    ignore:
+      - goos: linux
+        goarch: arm64
+      - goos: windows
+        goarch: arm64
+    ldflags:
+      - -s -w -extldflags "-fno-PIC -static" -X main.version={{.Version}} -X main.pkgName=nginx-meshctl
+    main: ./cmd/nginx-meshctl/
+    binary: nginx-meshctl
+
+changelog:
+  skip: true
+
+checksum:
+  name_template: checksums.txt
+
+sboms:
```

Comment:
If you want to generate SBOMs you need to add `anchore/sbom-action/download-syft` to the workflow to download `syft`

Reply:
Understood. I was planning on having a followup PR that included all the release related things.

Reply:
Unless something will break by having this directive in the goreleaser config without it.

Reply:
It will not break now, but at release time. So I think we're good if you're planning on addressing it later.